core/utils.js

```diff
@@ -308,14 +308,13 @@ Blockly.isRightButton = function(e) {
  * The origin (0,0) is the top-left corner of the Blockly svg.
  * @param {!Event} e Mouse event.
  * @param {!Element} svg SVG element.
+ * @param {SVGMatrix} matrix Inverted screen CTM to use.
```

Comment:
One concern here is that the matrix could be null (it starts out null in workspace_svg).  I suppose in that case I should fall back to getting the matrix directly from the svg.